README.md

```diff
@@ -157,10 +157,8 @@ make start-all
 Upload your service's trigger and submission contracts. The trigger contract is where WAVS will watch for events, and the submission contract is where the AVS service operator will submit the result on chain.
 
 ```bash docci-delay-per-cmd=1
-# deploy just the trigger to chain 2 (no eigenlayer here)
-forge script ./script/Deploy.s.sol 0x0000000000000000000000000000000000000000 false --sig "run(string,bool)" --rpc-url http://localhost:8645 --broadcast
-export SERVICE_TRIGGER_ADDR=`make get-trigger-from-deploy`
-
+# using base wrapped ETh contract -- https://basescan.org/address/0x4200000000000000000000000000000000000006
+export SERVICE_TRIGGER_ADDR="0x4200000000000000000000000000000000000006"
```

Comment:
or deploy your own contract like before, just with the base RPC url. i.e. anywhere this demo has `http://localhost:8645` you sub for that